hypervisor/include/dm/vpci.h

```diff
@@ -75,7 +75,7 @@ struct msi {
 struct msix {
 	struct msix_table_entry tables[CONFIG_MAX_MSIX_TABLE_NUM];
 	uint64_t  mmio_gpa;
-	uint64_t  mmio_hva;
+	void *mmio_hva;
 	uint64_t  mmio_size;
```

Comment:
If we change msix.mmio_hva to void *, are we sure we can do (void * + n) operation correctly?
        hva = vdev->msix.mmio_hva + vdev->msix.table_offset;

Reply:
Yes, and this is allowed in GCC extension: 

6.23 Arithmetic on void- and Function-Pointers
In GNU C, addition and subtraction operations are supported on pointers to void and on pointers to functions. This is done by treating the size of a void or of a function as 1.

But for the MISRA-C result, I will double check with Huihuang and Junjie.